specification/keyvault/resource-manager/Microsoft.KeyVault/stable/2019-09-01/providers.json

```diff
@@ -152,6 +158,29 @@
           "description": "Blob duration of specification."
         }
       }
+    },
+    "ProviderError": {
+      "description": "An error response from Key Vault resource provider",
+      "properties": {
+        "error": {
```

Comment:
Why is the actual error nested? Is this common of mgmt plane?

Reply:
I believed so. Both Key.json and KeyVault.json use the same model.